gamemodes/terrortown/gamemode/client/cl_help.lua

```diff
@@ -374,463 +24,342 @@ local function MuteTeamCallback(cv, old, new)
-	form:AddItem(cm)
-
-	form:CheckBox(GetTranslation("set_cross_gap_enable"), "ttt_crosshair_gap_enable")
-
-	local cb = form:NumSlider(GetTranslation("set_cross_gap"), "ttt_crosshair_gap", 0, 30, 0)
-	if cb.Label then
-		cb.Label:SetWrap(true)
-	end
-
-	cb = form:NumSlider(GetTranslation("set_cross_opacity"), "ttt_crosshair_opacity", 0, 1, 1)
-	if cb.Label then
-		cb.Label:SetWrap(true)
-	end
-
-	cb = form:NumSlider(GetTranslation("set_ironsight_cross_opacity"), "ttt_ironsights_crosshair_opacity", 0, 1, 1)
-	if cb.Label then
-		cb.Label:SetWrap(true)
-	end
-
-	cb = form:NumSlider(GetTranslation("set_cross_brightness"), "ttt_crosshair_brightness", 0, 1, 1)
-	if cb.Label then
-		cb.Label:SetWrap(true)
-	end
+function HELPSCRN:ShowMainMenu()
+	local frame = self.menuFrame
 
-	cb = form:NumSlider(GetTranslation("set_cross_size"), "ttt_crosshair_size", 0.1, 3, 1)
-	if cb.Label then
-		cb.Label:SetWrap(true)
+	-- IF MENU ELEMENT DOES NOT ALREADY EXIST, CREATE IT
+	if IsValid(frame) and not frame:IsFrameHidden() then
+		frame:ClearFrame(nil, nil, "help_title")
+	else
+		frame = vguihandler.GenerateFrame(width, height, "help_title", true)
 	end
 
-	cb = form:NumSlider(GetTranslation("set_cross_thickness"), "ttt_crosshair_thickness", 1, 10, 0)
-	if cb.Label then
-		cb.Label:SetWrap(true)
-	end
+	self.menuFrame = frame
 
-	cb = form:NumSlider(GetTranslation("set_cross_outlinethickness"), "ttt_crosshair_outlinethickness", 0, 5, 0)
-	if cb.Label then
-		cb.Label:SetWrap(true)
-	end
+	-- INIT MAIN MENU SPECIFIC STUFF
+	frame:SetPadding(self.padding, self.padding, self.padding, self.padding)
 
-	form:CheckBox(GetTranslation("set_cross_disable"), "ttt_disable_crosshair")
-	form:CheckBox(GetTranslation("set_minimal_id"), "ttt_minimal_targetid")
-	form:CheckBox(GetTranslation("set_cross_static_enable"), "ttt_crosshair_static")
-	form:CheckBox(GetTranslation("set_cross_dot_enable"), "ttt_crosshair_dot")
-	form:CheckBox(GetTranslation("set_cross_lines_enable"), "ttt_crosshair_lines")
-	form:CheckBox(GetTranslation("set_cross_weaponscale_enable"), "ttt_crosshair_weaponscale")
+	-- MARK AS MAIN MENU
+	self.currentMenuId = MAIN_MENU
 
-	cb = form:CheckBox(GetTranslation("set_lowsights"), "ttt_ironsights_lowered")
-	cb:SetTooltip(GetTranslation("set_lowsights_tip"))
+	-- MAKE MAIN FRAME SCROLLABLE
+	local scrollPanel = vgui.Create("DScrollPanelTTT2", frame)
+	scrollPanel:Dock(FILL)
 
-	form:Dock(FILL)
-end
+	-- SPLIT FRAME INTO A GRID LAYOUT
+	local dsettings = vgui.Create("DIconLayout", scrollPanel)
+	dsettings:Dock(FILL)
+	dsettings:SetSpaceX(self.padding)
+	dsettings:SetSpaceY(self.padding)
 
----
--- Creates the damage indicator settings for the help screen
--- @param Panel parent
--- @realm client
--- @internal
-function HELPSCRN:CreateDamageIndicatorSettings(parent)
-	local form = vgui.Create("DForm", parent)
-	form:SetName(GetTranslation("f1_dmgindicator_title"))
+	-- GENERATE MENU CONTENT
+	local menuTbl = {}
+	local helpData = HELP_MENU_DATA:BindData(menuTbl)
```

Comment:
To be honest, a `.` would work as well here. But since the "subobjects" have multiple instances, I used the same "style" here as well to keep it simple